pokemongo_bot/__init__.py

```diff
@@ -548,6 +557,7 @@ def _register_events(self):
     def tick(self):
         self.health_record.heartbeat()
         self.cell = self.get_meta_cell()
+        inventory.update_web_inventory()
```

Comment:
I changed that. It now exports from cached inventory instead of making an api call.